linkedList.js

```diff
@@ -3,6 +3,96 @@
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
-function linkedListGenerator(){
+function linkedListGenerator() {
+  let head = null;
+  let tail = null;
 
-}
\ No newline at end of file
+  let getHead = function () {
+    return head;
+  };
+  let getTail = function () {
+    return tail;
+  }
+  let add = function (x) {
+    let newNode = {};
+    newNode.value = x;
+    newNode.next = null;
+    if (head === null) {
+      head = newNode;
+      tail = newNode;
+    } else {
+      tail.next = newNode;
+      tail = newNode;
+    }
+    return newNode;
+  }
+  let remove = function (x) {
+    if (x === 0) {
```

Comment:
Insert newline above.

And you probably want to handle the case where if you try to delete from index 0 with an empty linked list.